src/server/ocpp/services/OCPPService.ts

```diff
@@ -1170,6 +1170,7 @@ export default class OCPPService {
       } else {
         consumption.consumption = transaction.currentConsumptionWh;
         consumption.instantPower = Math.round(transaction.currentConsumption);
+        consumption.instantAmps = transaction.currentConsumption / 230; // Utils convert method was updated. Needs to be handled by Serge.
```

Comment:
It's not that hard to just merge with the master-qa branch and follow the API change of the method: it's just one argument added to the conversion function.